clusterbuster

```diff
@@ -1185,7 +1195,16 @@ function ts() {
 }
 
 function run_status_monitor() {
-    (while : ; do sleep 1; printf '++Mark++ ++Mark+ %(%s)T%s' -1 $'\n'; done &); ____OC get pod "$@" -w -o jsonpath='{.metadata.namespace} {.metadata.name} {.status.phase}{"\n"}'
+    function run_status_monitor_1() {
```

Comment:
What is the meaning **run_status_monitor_1** of 1 ?

Reply:
It's a convention I use to basically indicate that the function is a private helper of the function with the _1 stripped off (i. e. a private helper of run_status_monitor).